test-directory/tests.sh

```diff
@@ -1,3 +1,9 @@
 #!bin/sh
 emacs --batch --eval '(message (pp (+ 2 2)))'
```

Comment:
Question: is it normal that the output of this line does not show up in the Travis CI log? maybe there's a bug with the docker command…

https://travis-ci.com/pfitaxel/learn-ocaml.el/builds/114446444#L463

Reply:
I was thinking on that may be i have to put sh -c in the call

Reply:
no

Reply:
i found the error, when runing the container i didn't update the location of tests.sh. It didn't find it and created a directory inside the container. And when you execute sh in a directory called test.sh it does nothing

Reply:
ok